test/local.tests.ts

```diff
@@ -264,7 +266,7 @@ int Add(int x, int y)
         });
 
         it("local function declaration with extern modifier", async () => {
-            const input = Input.InClass(`extern static void Foo() { }`);
```

Comment:
I was wondering why this test already exists since 3 years ago and passes, but it still doesn't work in vscode.

Turns out the test was cheating with `InClass` for a local function...